### PR TITLE
remove unused code

### PR DESCRIPTION
**Pull request by michaelkirk**
MapboxCoreNavigation/CoreFeedbackEvent.swift is a vestige of the telemetry stuff which was removed in several commits, including 62e6415d6e805c5b4696fbd439615144096f6f01

### CODE REVIEW

File: MapboxCoreNavigation/MMEEventsManager.swift [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import AVFoundation
 import MapboxDirections
-import Polyline
```
Comment (michaelkirk) [inline MapboxCoreNavigation/MMEEventsManager.swift:3]: Unused import.

File: MapboxCoreNavigation/RouteController.swift [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import CoreLocation
 import Foundation
 import MapboxDirections
-import Polyline
```
Comment (michaelkirk) [inline MapboxCoreNavigation/RouteController.swift:4]: Unused import.

**Review by ianthetechie — APPROVED** [review]
LGTM; thanks!

**Comment by ianthetechie** [comment]
Oh I guess I'm not a write access contributor so that doesn't count for much :P Any objections from you @boldtrn?

**Review by hactar — APPROVED** [review]
Thanks!

**Pull request merged**